custom_components/healthchecksio/config_flow.py

```
@@ -109,34 +110,115 @@ def _clean_url(url: str) -> str:
+                default=user_input.get(
+                    CONF_SELF_HOSTED, fallback.get(CONF_SELF_HOSTED, DEFAULT_SELF_HOSTED)
+                ),
+            ): selector.BooleanSelector(selector.BooleanSelectorConfig()),
+        }
+    )
+
+
+def _build_self_hosted_schema(
+    user_input: MutableMapping[str, Any] | None,
+    fallback: Mapping[str, Any] | None = None,
+    reconf: bool = False,
```

Pylint (3.3.7)

[warning] Unused argument 'reconf'

(W0613)